library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -283,20 +285,36 @@ private CallbackThrottleCreator getCallbackThrottleCreator(CallbackThrottleCreat
 
         @Override
         public Notification customNotificationFrom(NotificationCompat.Builder builder, DownloadBatchStatus payload) {
+            if (payload.status() == DownloadBatchStatus.Status.DELETION) {
+                return createDeletedNotification(builder, payload);
+            } else {
+                DownloadBatchTitle downloadBatchTitle = payload.getDownloadBatchTitle();
+                int percentageDownloaded = payload.percentageDownloaded();
+                int bytesFileSize = (int) payload.bytesTotalSize();
+                int bytesDownloaded = (int) payload.bytesDownloaded();
+                String title = downloadBatchTitle.asString();
+                String content = percentageDownloaded + "% downloaded";
+
+                return builder
+                        .setProgress(bytesFileSize, bytesDownloaded, NOT_INDETERMINATE)
+                        .setSmallIcon(notificationIcon)
+                        .setContentTitle(title)
+                        .setContentText(content)
+                        .build();
+
+            }
+        }
+
+        private Notification createDeletedNotification(NotificationCompat.Builder builder, DownloadBatchStatus payload) {
             DownloadBatchTitle downloadBatchTitle = payload.getDownloadBatchTitle();
-            int percentageDownloaded = payload.percentageDownloaded();
-            int bytesFileSize = (int) payload.bytesTotalSize();
-            int bytesDownloaded = (int) payload.bytesDownloaded();
             String title = downloadBatchTitle.asString();
-            String content = percentageDownloaded + "% downloaded";
+            String content = "Deleted";
 
             return builder
-                    .setProgress(bytesFileSize, bytesDownloaded, NOT_INDETERMINATE)
                     .setSmallIcon(notificationIcon)
```

Comment:
This code is also reused in the else, shall we try to reuse as much as possible?

Reply:
Can do 👍

Reply:
2b040c5